content/docs/1.6.0/deploy/install/install-with-fleet.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Install with Fleet
+weight: 10
+---
+
+In this section, you will learn how to install Longhorn with Fleet.
+
+### Prerequisites
+
+- Each node in the Kubernetes cluster where Longhorn will be installed must fulfill [these requirements.](../#installation-requirements)
+- Helm v2.0+ must be installed on your workstation.
+  - 1. Refer to the official documentation for help installing Helm.
+  - 2. If you're using a Helm version prior to version 3.0, you need to [install Tiller in your Kubernetes cluster with role-based access control (RBAC)](https://v2.helm.sh/docs/using_helm/#tiller-namespaces-and-rbac).
+
+> [This script](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/scripts/environment_check.sh) can be used to check the Longhorn environment for potential issues.
+
+### Installing Fleet
+
+1. Install [Fleet](https://fleet.rancher.io/) using Helm. Add the Helm repository and install Fleet:
+
+    ```bash
+    helm repo add fleet https://rancher.github.io/fleet-helm-charts/
+    helm -n cattle-fleet-system install --create-namespace --wait fleet-crd fleet/fleet-crd
+    helm -n cattle-fleet-system install --create-namespace --wait fleet fleet/fleet
+    ```
+
+    Wait for Fleet components to be deployed in the cattle-fleet-system namespace.
+
+### Installing Longhorn
+
+1. In your Gitops repository, create [fleet.yaml](https://fleet.rancher.io/ref-fleet-yaml) to configure Longhorn installation:
+
+    Ensure the fleet.yaml file includes the following configuration to install Longhorn in longhorn-system namespace:
```

Comment:
```suggestion
    - Parameter for installing Longhorn in the `longhorn-system` namespace
```

Reply:
I used an unordered list to organize the code blocks. There are three list items.